tests/functional/test_measurements.py

```diff
@@ -0,0 +1,96 @@
+        # a cyclic voltammagram and indexing, and append them:
+        cv = ec_measurement.as_cv()
+        cvs_1_plus_2 = cv[1] + cv[2]
+
+        # Check that the calibration survived all that:
+        assert cvs_1_plus_2.RE_vs_RHE == ec_measurement.RE_vs_RHE
+        # Check that the main time variable, corresponding to potential, wasn't corrupted:
+        assert len(cvs_1_plus_2.grab("potential")[0]) == len(
+            cvs_1_plus_2["time/s"].data
+        )
+        # Check that the selector is still available and works with the main time variable:
+        assert len(cvs_1_plus_2.selector.data) == len(cvs_1_plus_2.t)
+
+    def test_calibration_over_save_load(self, composed_measurement):
+        """Test save and load functionality"""
+        # Now let's try calibrating just the loaded one:
+        composed_measurement_copy = Measurement.from_dict(
+            composed_measurement.as_dict()
+        )
+
+        RE_vs_RHE = 1
+        A_el = 2
+        composed_measurement_copy.calibrate(RE_vs_RHE=RE_vs_RHE, A_el=A_el)
+
+        # First we check that its v attribute is calibrated by comparing to the grabbed
+        # uncalibrated potential:
+        assert (
+            composed_measurement_copy.v[0]
+            == composed_measurement.grab("raw_potential")[1][0] + RE_vs_RHE
+        )
+        # ^ note this only builds the potential for cv12_loaded
+
+        # First we check that its j attribute is calibrated by comparing to the same property
+        # of the original, still uncalibrated, measurement:
+        assert composed_measurement_copy.j[0] == composed_measurement.j[0] / A_el
+        # ^ note this builds the current for both cv12 and cv12_loaded
+
+        # As a result: there should be more cached series in cv12_loaded.
+        # However, the number of series in the series_list should be conserved:
+        assert len(composed_measurement.series_list) == len(
+            composed_measurement_copy.series_list
+        )
+
+        # Now, try copying the calibrated measurement by as_dict() and from_dict():
+        meas12_copied = Measurement.from_dict(composed_measurement_copy.as_dict())
+        # And check if it still has the calibration:
+        assert meas12_copied.A_el == A_el
+        # And that it can still apply it:
+        assert (
+            meas12_copied.grab("potential")[1][0]
+            == approx(composed_measurement.grab("potential")[1][0] + RE_vs_RHE)
+        )
+
+        # Finally, save and load the copied measurement and check that it's still there:
+        # FIXME the load save test add to the total test time, this one is a primary
+        # candidate to be removed if we properly trust the backend tests
+        i2 = meas12_copied.save()
+        meas12_copied_loaded = Measurement.get(i2)
+        assert meas12_copied_loaded.RE_vs_RHE == RE_vs_RHE
+        assert len(meas12_copied_loaded.v) == len(composed_measurement.v)
+
+    def test_to_and_as_dict(self, composed_measurement):
```

Comment:
"to_and_from" would be better English. "as_and_from" would mirror the method names `as_dict()` and `from_dict()`